content/release-notes/1.13.5.md

```diff
@@ -5,6 +5,6 @@ title: "1.13.5"
 kubernetes: "1.15, 1.16, and 1.17"
 ---
 
-{{<changes>}}
-* Support Bundle and Preflight have been updated to [v0.9.26](https://github.com/replicatedhq/troubleshoot/releases/tag/v0.9.26).
-{{</changes>}}
+{{<fixes>}}
+* Preflight outcomes generated using regex analyzers with less-than (`<`), less-than-or-equal (`<=`), or greater-than-or-equal (`>=`) conditions were almost always incorrect. Fixed by ensuring the operators strings correspond to the correct comparison operator. 
```

Comment:
`almost always incorrect` feels a little unclear, how about this:
 
```suggestion
* Preflight outcomes generated using regex analyzers with less-than (`<`), less-than-or-equal (`<=`), or greater-than-or-equal (`>=`) conditions were evaluated in reverse. Fixed by ensuring the operators strings correspond to the correct comparison operator. 
```

Reply:
Yeah, that's better